libs/components/split-view/testing/src/split-view/split-view-fixture.ts

```diff
@@ -7,7 +7,7 @@ import { SkySplitViewFixtureDrawer } from './split-view-fixture-drawer';
 import { SkySplitViewFixtureWorkspace } from './split-view-fixture-workspace';
 
 /**
- * Provides information for and interaction with a SKY UX split view component.
+ * Information for and interaction with a SKY UX split view component.
```

Comment:
I don't see this text in the split view docs on the live site. Do we need to file a separate bug to figure out why it's not showing up?

Also, based on the location within the file, it seems like we should keep "Provides" here because it seems like this is probably a section overview rather than a table entry for a property.

Reply:
@johnhwhite, do I need to file a bug since this doesn't show up in the docs? Or is this intentionally internal only?

Reply:
@blackbaud-johnly this file is also marked `@internal` — it's a test fixture within the split-view testing package and will be revisited as part of AB#2195542 to implement the newer harness pattern.